### PR TITLE
CDS-54919-Jira-update-step

### PR DESCRIPTION
**Pull request by roshnisarangadharan**
Added new screenshots, added a section for modifying issue type, self review.

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/continuous-delivery/cd-advanced/ticketing-systems-category/update-jira-issues-in-cd-stages.md [hunk 1]
```diff
@@ -122,28 +118,43 @@ For date and time fields:
...
+1. Enter an **Issue Key** in the Jira update step. 
+2. In the **Add Jira Fields** dialog, select **Issue Type**, then select **Add**. The **Issue Type** field appears under **Optional Configuration**.
+   
+   ![](static/modify-issue-type.png)
+   
+3. Select a new issue type to modify the Issue Type of the selected Jira issue, and select **Apply Changes**. 
+
+  ![](static/update-issue-type.png)
+
+The issue type for the selected Jira issue is now updated with the issue type you selected.
```
Comment (naman-goenka) [inline docs/continuous-delivery/cd-advanced/ticketing-systems-category/update-jira-issues-in-cd-stages.md:134]: `Issue`

File: docs/continuous-delivery/cd-advanced/ticketing-systems-category/update-jira-issues-in-cd-stages.md [hunk 2]
```diff
@@ -27,75 +27,71 @@ The following video shows you how to use the Jira Create, Jira Update, and Jira
...
+![](static/status-and-transition.png)
 
-![](./static/update-jira-issues-in-cd-stages-17.png)
+If the issue is not part of a Jira workflow and does not have transition options, then the step will fail. For more information, go to [Statuses and transitions](https://support.atlassian.com/jira-cloud-administration/docs/work-with-issue-workflows/#Workingwithworkflows-steps) from Atlasssian.
 
-If the issue is not part of a Jira workflow and does not have transition options, then the step will fail.See [Statuses and transitions](https://support.atlassian.com/jira-cloud-administration/docs/work-with-issue-workflows/#Workingwithworkflows-steps) from Atlasssian.
+### Add issue fields
 
-## Step 2: Add Issue Fields
+You can select specific fields to update within a Jira issue. For more information, go to [Jira Custom Fields](../../../first-gen/continuous-delivery/model-cd-pipeline/workflows/jira-integration.md#jira-custom-fields).
```
Comment (naman-goenka) [inline docs/continuous-delivery/cd-advanced/ticketing-systems-category/update-jira-issues-in-cd-stages.md:88]: this links to CG

**Review by naman-goenka — APPROVED** [review]

File: docs/continuous-delivery/cd-advanced/ticketing-systems-category/update-jira-issues-in-cd-stages.md (hunk 2 above)
Comment (roshnisarangadharan) [reply docs/continuous-delivery/cd-advanced/ticketing-systems-category/update-jira-issues-in-cd-stages.md:88]: Will check this one. Looks like we don't have this topic for NG.

**Review by aimurphy — DISMISSED** [review]
lgtm

**Pull request merged**